miso-web/src/main/webapp/scripts/list.js

```diff
@@ -33,6 +33,208 @@ ListUtils = (function() {
+      help: "Checks when any person last edited this item.  For rules about dates, see below."
+    },
+    "creator": {
+      term: "creator:USER",
+      help: "Check for items entered into MISO by a particular user.  For the rules about users, see below."
+    },
+    "changedby": {
+      term: "changedby:USER",
+      help: "Checks for the last person to edit this item in MISO.  For the rules about users, see below."
+    },
+    "platform": {
+      term: "platform:PLATFORM",
+      help: "Check if this item is meant for a particular platform type: ILLUMINA, LS454, SOLID, IONTORRENT, PACBIO, OXFORDNANOPORE."
+    },
+    "index_name": {
+      term: "index:NAME",
+      help: "Checks if this item has the index provided.  The name can be a partial match."
+    },
+    "index_seq": {
+      term: "index:SEQ",
+      help: "Checks if this item has the index provided.  The sequence must be an exact match."
+    },
+    "class": {
+      term: "class:NAME",
+      help: "Check if the item belong to the sample class provided.  This is taken as a partial match."
+    },
+    "institute": {
+      term: "institute:NAME",
+      help: "Check if the item came from the institute mentioned.  This is a partial match."
+    },
+    "external": {
+      term: "external:NAME",
+      help: "Checks if an item came from the external identifier or external name."
+    },
+    "box": {
+      term: "box:NAME",
+      help: "Checks if an item is located in a particular box.  The name can either be the partial name or partial alias of the box."
+    }
+  };
+
+  var makeTooltipHelp = function(target) {
+    return "Search syntax: \
+      <br/> \
+      <br/>"
+      + target.searchTermSelector(searchTerms).map(function(term) {
+        return term["term"];
+      }).join('<br/>');
+  };
+
+  var makeSearchTooltip = function(searchDivId, target) {
+    var searchTooltipId = "searchHelpTooltip_" + target.name.replace(/\s/g, '');
+
+    jQuery("#" + searchDivId).append(' \
+      <div id="' + searchTooltipId + '" class="tooltip"> \
+        <span> \
+          <img id="searchHelpQuestionMark" src="/styles/images/question_mark.png" \
+        </span> \
+        <span class="tooltiptext"> \
+          ' + makeTooltipHelp(target) + ' \
+        </span> \
+      </div>');
+
+    return searchTooltipId;
+  };
+
+  var makePopupTableBody = function(target) {
+    var result = "";
+    var targetTerms = target.searchTermSelector(searchTerms);
+
+    for (var i = 0; i < targetTerms.length; ++i) {
+      result += "<tr><td>" + targetTerms[i]["term"] + "</td><td>" + targetTerms[i]["help"] + "</td></tr>";
+    }
+
+    return result;
+  };
+
+  var makePopupTable = function(target) {
+    return ' \
+      <table class="searchHelpTable">\
+        <caption><h2>Search Terms</h2></caption>\
+        <thead> \
+          <tr><th>Syntax</th><th>Meaning</th></tr> \
+        </thead> \
+      ' + makePopupTableBody(target) + ' \
+      </table>';
+  };
+
+  var dateGrammar = '\
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use string concatenation rather than line continuation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AMultilineStringLiterals)